src/memphis.njk

```diff
@@ -52,12 +52,12 @@
                   Epicenter
                 </div>
                 <div class="col-md-8 text-right d-flex justify-content-center justify-content-md-end">
-                  <a class="btn btn-primary mr-2 px-3" href="#"><i class="fas fa-globe"></i></a>
-                  <a class="btn btn-primary mr-2 px-3" href="#"><i class="fab fa-facebook"></i></a>
-                  <a class="btn btn-primary mr-2 px-3" href="#"><i class="fab fa-instagram"></i></a>
-                  <a class="btn btn-primary px-3" href="#"><i class="fab fa-twitter"></i></a>
+                  <a class="btn btn-primary mr-2 px-3" href="https://www.epicentermemphis.org/" aria-label="Navigation link to Epicenter web page"><i class="fas fa-globe"></i></a>
+                  <a class="btn btn-primary mr-2 px-3" href="#" aria-label="Navigation link to Epicenter Facebook"><i class="fab fa-facebook"></i></a>
+                  <a class="btn btn-primary mr-2 px-3" href="https://www.instagram.com/epicentermemphis/" aria-label="Navigation link to Epicenter instagram"><i class="fab fa-instagram"></i></a>
+                  <a class="btn btn-primary px-3" href="https://twitter.com/epimemphis" aria-label="Navigation link to Epicenter twitter"><i class="fab fa-twitter"></i></a>
                 </div>
-              </div>
+              </div> 
```

Comment:
nit: whitespace

Reply:
I couldn't find this one

Reply:
this kind of a diff means that in your copy, there is a space at the end of this line. It's okay if it is here, but it is distraction, which is why it is preferred to not adjust trailing whitespace unless it is impactful to the markup. We can leave it in if you are having trouble finding it.